scripts/dataset_processing/tts/hui_acg/phonemizer_local.py

```diff
@@ -0,0 +1,78 @@
+"""
+The original JSON splits generated from `get_data.py` only contain graphemes inputs. We recommend adding phonemes as
+well to obtain better quality of synthesized audios. So you would expect the dataset double sized. This script implements
+such idea. Before running, please install dependency on your local machine as shown below. More details are described in
+https://github.com/NVIDIA/NeMo/blob/main/tutorials/tts/Fastpitch_Training_GermanTTS.ipynb
+
+$ pip install phonemizer && apt-get install espeak-ng
+
+Usage for example:
+$ python scripts/dataset_processing/tts/hui_acg/phonemizer_local.py \
+    --preserve-punctuation \
+    --json-manifests ~/tmp/val_manifest_text_normed.json ~/tmp/test_manifest_text_normed.json
+"""
+
+import argparse
+import json
+from pathlib import Path
+
+from phonemizer.backend import EspeakBackend
+from tqdm import tqdm
+
+from nemo.utils import logging
+
+
+def get_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter, description="Add phonemes in JSON manifests."
+    )
+    parser.add_argument(
+        "--json-manifests",
+        nargs="+",
+        type=Path,
+        help="Specify a full path of a JSON manifest. You could add multiple manifest.",
+    )
+    parser.add_argument(
+        "--preserve-punctuation",
+        default=False,
+        action='store_true',
+        help="Preserve punctuations if True when converting char into phonemes.",
+    )
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = get_args()
+    input_manifest_filepaths = args.json_manifests
+    preserve_punctuation = args.preserve_punctuation
+    # TODO (xueyang): consider with_stress=True and add stress symbols into charset of tokenizer.
```

Comment:
Remove all TODO comments.

Reply:
will do.

Reply:
a second thought is that mixing with phonemes is still experimental so that it would be helpful to leave the `TODO` here for the revision in next iterations.